lib/api.rb

```diff
@@ -0,0 +1,7 @@
+module Api
+  class AuthenticationError < StandardError; end
+  class Forbidden < StandardError; end
```

Comment:
Not sure I like the word Forbidden (and NotFound) just bare without an Error or something at the end.